content/admin/org_settings/API-credentials.md

```diff
@@ -0,0 +1,27 @@
+<!--
+title: "Security API Keys"
+description: "About article for the API Keys"
+tags: "Admin system settings security API keys configuration"
+-->
+
+When you download the Contrast JVM plugin (also called the Engine), it comes pre-fitted with a set of randomly generated credentials for your organization that don't involve the passwords of any users in the organization. When the plugin communicates with the Contrast application, it authenticates using these credentials. 
+ 
+<a href="assets/images/API-credentials-1.png" rel="lightbox" title="Default authentication"><img class="thumbnail" src="assets/images/API-credentials-1.png"/></a>
+
+## How It Works 
+
+Contrast added another layer of security through an organization API key that you can enable and manage on your own. In the case of a security breach, an unauthorized user can't submit forged or malicious data to your organization because their organization API key is wrong. With the API key enabled, authentication follows the process shown in the image below.
```

Comment:
API Key is always enabled, and there is not way to turn it off. 

Probably you could remove "you can enable" and "With the API key enabled"